rest-api/tools/deploy_app.sh

```diff
@@ -154,7 +154,7 @@ then
 
   if [ "$TARGET" == "app" ] || [ "$TARGET" == "all" ]
   then
-    if [ "${PROJECT}" = "all-of-us-rdr-prod" ]
+    if [ "${PROJECT}" = "all-of-us-rdr-prod" ] || [ "${PROJECT}" = "all-of-us-rdr-staging" ]
```

Comment:
I'm not sure I'd merge this into master, as we don't want staging normally to use app_prod.yaml... only during load tests.

If we do want to merge something into master here, we should probably add a --use_prod_yaml flag that we check here instead.

Reply:
done